examples/test_case_management_v1_examples.py

```diff
@@ -109,7 +109,7 @@ def test_create_case_example(self):
                 severity=4,
             ).get_result()
 
-            print(json.dumps(case, indent=2))
+            print('\ncreate_case() result:\n', json.dumps(case, indent=2))
```

Comment:
Concatenating strings in Python's `print` function with comma (,) adds a space between them. This brokes the printed result a little bit:
```
upload_file() result:
 {
  "id": "xxx",
  "filename": "example.log",
  "size_in_bytes": 42,
  "created_at": "2021-03-23 16:52:19",
  "url": "xxx"
}
```
```suggestion
            print('\ncreate_case() result:\n' + json.dumps(case, indent=2))
```

Reply:
I like @pyrooka 's suggestion :)